packages/genesys-spark-components/src/components/stable/gux-context-menu/gux-context-menu.scss

```diff
@@ -7,26 +7,20 @@
 }
 
 gux-icon {
-  width: var(--gse-ui-icon-size-sm); // TODO: [COMUI-2290] missing token
-  height: var(--gse-ui-icon-size-sm); // TODO: [COMUI-2290] missing token
+  width: var(--gse-ui-icon-size-sm);
+  height: var(--gse-ui-icon-size-sm);
   color: var(--gse-ui-button-ghost-default-foregroundColor);
 }
 
 button {
-  height: var(
-    --gse-ui-button-default-height
-  ); // TODO: [COMUI-2290] name should be more specific
-
-  padding: 0 8px; // TODO: [COMUI-2290] missing token
+  height: var(--gse-ui-button-default-height);
+  padding: 0 8px;
   border-radius: var(--gse-ui-button-borderRadius);
 }
 
 button.gux-compact {
-  height: var(
-    --gse-ui-contextMenu-button-compact
-  ); // TODO: [COMUI-2290] name should be more specific
-
-  padding: 0 4px; // TODO: [COMUI-2290] missing token
+  height: var(--gse-ui-button-compact-height);
+  padding: 0 4px;
```

Comment:
Open question in the ticket comments if tokens should be used for the padding.

Reply:
I think they should be.

Reply:
+1, we shouldn't have any hardcoded `px` values.

Reply:
New tokens have been added!